ads/_config.js

```diff
@@ -48,6 +49,11 @@ let AdNetworkConfigDef;
  *   // We highly recommend all networks to implement the API,
  *   // see details in the README.md
  *   renderStartImplemented: boolean
+ *
+ *   // The width / height ratio for full width ad units.
+ *   // If absent, it means the network does not support full width ad units.
+ *   // Example value: 1.2
+ *   fullWidthHeightRatio: number
```

Comment:
if you want to force a range for this number, i suggest you add it to the documentation of this feature (which can be a separate PR).

Reply:
Sounds good - haven't included it in this PR.